build/progressionMeta.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
normal progressionDefs from destiny don't give us nice names... maybe it's in another table? either way we need special scales for each one and bg colors...